Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -233,12 +99,12 @@ class DataDecoderTask
       }
 
       gsl::span<const std::byte> buffer(reinterpret_cast<const std::byte*>(raw), sizeof(o2::header::RAWDataHeaderV4) + payloadSize);
-      decodeBuffer(buffer, digits);
+      mDecoder.decodeBuffer(buffer);
     }
   }
 
   //_________________________________________________________________________________________________
-  void decodeReadout(const o2::framework::DataRef& input, std::vector<o2::mch::Digit>& digits)
+  void decodeReadout(const o2::framework::DataRef& input)
```

Comment:
now I am confused : what is the "readout" input ? Please add some doc

Reply:
I have added some comments as well as an example in the README.